File: platform/sdk/nodejs/examples/cloud/cloud-010_copy_object.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("src-bucket");
client.createBucket("dst-bucket");
client.putObject("src-bucket", "src.txt", "copy-me");
const result = client.copyObject("src-bucket", "src.txt", "dst-bucket", "dst.txt");
if (!result.ok) process.exit(1);
console.log("XJ-OK: cloud-010_copy_object");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-010_copy_object.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("src-bucket")
client.createBucket("dst-bucket")
client.putObject("src-bucket", "src.txt", "copy-me")
result = client.copyObject("src-bucket", "src.txt", "dst-bucket", "dst.txt")
if not result["ok"]:
    sys.exit(1)
print("XJ-OK: cloud-010_copy_object")
sys.exit(0)
